package.json

```diff
@@ -40,21 +40,19 @@
     "lint-markdown-fix": "yarn lint-markdown --fix",
     "lint-markdown": "markdownlint \"**/*.md\" --config=.markdownlint.json --ignore-path=.markdownlintignore",
     "lint-stylelint": "npx nx lint website stylelint",
-    "lint": "npx nx lint eslint-plugin --skip-nx-cache && npx nx run-many --target=lint --parallel --exclude eslint-plugin",
+    "lint": "npx nx lint eslint-plugin --skip-nx-cache && npx nx run-many --target=lint --parallel --exclude eslint-plugin && eslint . --ignore-pattern=packages --cache",
```

Comment:
This still does the `&&` strategy 😕. Surely there's got to be a way in Nx to set up a project to include root-level files, so the `npx nx run-many --target=lint` run would include them?

Reply:
Sure thing, I can look into that